src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsData.cs

```diff
@@ -0,0 +1,148 @@
+
+            return retTensorsData;
+        }
+
+        /// <summary>
+        /// Gets the number of Tensor in TensorsData class
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
+        public int Count
+        {
+            get {
+                if (_count != Tensor.InvalidCount)
+                    return _count;
+
+                NNStreamerError ret = NNStreamerError.None;
+                ret = Interop.Util.GetTensorsCount(_handle, out int count);
+                NNStreamer.CheckException(ret, "unable to get the count of TensorsData");
+
+                _count = count;
+                return _count;
+            }
+        }
+
+        /// <summary>
+        /// Sets a tensor data to given index.
+        /// </summary>
+        /// <param name="index">The index of the tensor.</param>
+        /// <param name="buffer">Raw tensor data to be set.</param>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 5.5 </since_tizen>
+        public void SetTensorData(int index, byte[] buffer)
+        {
+            NNStreamerError ret = NNStreamerError.None;
+
+            ret = Interop.Util.SetTensorData(_handle, index, buffer, buffer.Length);
+            NNStreamer.CheckException(ret, "unable to set the buffer of TensorsData: " + index.ToString());
+        }
+
+        /// <summary>
+        /// Gets a tensor data to given index.
+        /// </summary>
+        /// <param name="index">The index of the tensor.</param>
+        /// <returns>Raw tensor data</returns>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 5.5 </since_tizen>
+        public byte[] GetTensorData(int index)
+        {
+            byte[] retBuffer;
+            IntPtr raw_data;
+            int size;
+
+            NNStreamerError ret = NNStreamerError.None;
+            ret = Interop.Util.GetTensorData(_handle, index, out raw_data, out size);
+            NNStreamer.CheckException(ret, "unable to get the buffer of TensorsData: " + index.ToString());
+
+            retBuffer = Interop.Util.IntPtrToByteArray(raw_data, size);
+            return retBuffer;
+        }
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object.
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object. Can also dispose any other disposable objects.
+        /// </summary>
+        /// <param name="disposing">If true, disposes any disposable objects. If false, does not dispose disposable objects.</param>
+        protected virtual void Dispose(bool disposing)
```

Comment:
https://docs.microsoft.com/ko-kr/dotnet/standard/garbage-collection/implementing-dispose

_handle is umanaged object

Reply:
I understood. I will.